programmers/난이도별/level02.구명보트/sjjyy.java

```diff
@@ -0,0 +1,31 @@
+package 난이도별.level02.구명보트;
+
+import java.util.Arrays;
+
+public class sjjyy {
+    public static int solution(int[] people, int limit) {
+        int answer = people.length;
+
+        Arrays.sort(people);
+        int j = people.length - 1;
+
+        for (int i = 0 ; i < j ;) {
+            if (people[i] + people[j] <= limit) {
+                answer--;
+                i++;
+            }
+            j--;
+        }
```

Comment:
```suggestion
        int j = people.length - 1, i = 0;

        while (i < j) {
            if (people[i] + people[j] <= limit) {
                answer--;
                i++;
            }
            j--;
        }
```
이럴땐 while로 표현하는게 더 좋을 것 같아요! 이게 맞다라기보단 그냥 저의 주관적인 생각입니다~

Reply:
아 while을 쓸 생각을 못하고 for문으로 이리저리 굴리고 있었네요 😅😅 감사합니다!